test/unit/math/gpu/add_test.cpp

```diff
@@ -13,7 +13,7 @@ TEST(MathMatrixGPU, add_v_exception_pass) {
   stan::math::matrix_gpu d11(d1);
   stan::math::matrix_gpu d22(d2);
   stan::math::matrix_gpu d33(3, 1);
-  EXPECT_NO_THROW(d33 = stan::math::add(d11, d22));
+  EXPECT_NO_THROW(d33 = d11 + d22);
```

Comment:
A different path here might be to just add a test that shows that `+` calls `add` for all the right types...  There are pros and cons to each. Cons to this approach replacing all `add`s with `+`:
1) the Stan language actually generates code that calls `add`, not `+`, so in some sense these are testing a non-primary API
2) No longer mirrors the other `add_test.cpp`s
3) You had to change more code, but that's basically done now.

Reply:
> the Stan language actually generates code that calls add, not +, so in some sense these are testing a non-primary API

While it's testing a non-primary api, the GPU stuff is also a non-primary API right? As in, all of the GPU functions will actually be called in their prim and rev counterparts, so even generated code (as of now) would not directly touch these APIs.

> expressing more of the API a Math library programmer would like to use

Yeah the main reason for adding these is so it will be a little easier to write out other things in prim and rev so they can use the GPU